iceoryx_utils/include/iceoryx_utils/internal/posix_wrapper/argv_inspection.hpp

```diff
@@ -44,7 +44,7 @@ class ArgvInspector
 
```

Comment:
since you deleted the corresponding cpp file, I guess this should also be deleted

Reply:
done